tests/conftest.py

```diff
@@ -17,3 +21,26 @@ def user():
     return User.objects.create_user(
         email="alice@test.com", password="password", name="Alice"
     )
+
+
+# set these values to have messages generated by tests appear in a test channel
+slack_token = os.environ.get("SLACK_BOT_TOKEN")
+slack_test_channel = os.environ.get("SLACK_TEST_CHANNEL")
+
+SlackMessage = namedtuple("SlackMessage", ("text", "channel"))
```

Comment:
A dataclass might be nicer here?

Reply:
I chose namedtuple as it preserves [0]/[1] access as if it was the `*args` passed to the function, which I think is more intuitive.